java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3649,7 +3649,11 @@ public void xgroupCreateConsumer_xgroupDelConsumer_xreadgroup_xack(BaseClient cl
         assertNotNull(streamid_3);
 
         // xack that streamid_1, and streamid_2 was received
-        assertEquals(2L, client.xack(key, groupName, new String[] {streamid_1, streamid_2}).get());
+        assertEquals(
+                2L,
+                client
+                        .xack(gs(key), gs(groupName), new GlideString[] {gs(streamid_1), gs(streamid_2)})
+                        .get());
```

Comment:
Please duplicate the test instead of leaving old `xack` untested - like you did for `setbit`.